app/src/main/res/layout/about_event_detail.xml

```diff
@@ -190,19 +190,23 @@
                         android:drawableLeft="@{ @drawable/ic_earth }"
                         android:drawablePadding="@dimen/spacing_small"
                         android:drawableStart="@{ @drawable/ic_earth }"
+                        android:text='@{ event.paymentCountry }'
+                        android:visibility="@{ (event.paymentCountry != null) ? View.VISIBLE : View.GONE }"/>
                         android:gravity="center_vertical"
-                        android:padding="@dimen/spacing_small"
-                        android:text='@{ event.paymentCountry }' />
+                        android:padding="@dimen/spacing_small" />
+
```

Comment:
Unnecessary changes. One line is needed to be added and several are reordered. This is undesirable as it increases entropy of the PR and we think there are a lot of changes which need to be reviewed. It also pollutes the git blame system where the original author is shadowed by the one rearranging the code. That's why it's a suggestion to only format files in a separate code format PR